examples/carbon/default.carbon

```diff
@@ -1,9 +1,11 @@
-package sample api;
+// To run this in Carbon Explorer you'll need to comment out the following line:
+// package sample api;
```

Comment:
it's already commented out.. so... uncomment to run in the non-explorer?

Reply:
lol yes! will fix